src/Draggable/Sensors/TouchSensor/tests/TouchSensor.test.js

```diff
@@ -0,0 +1,194 @@
+import {
+  createSandbox,
+  triggerEvent,
+  waitForDragDelay,
+  DRAG_DELAY,
+  touchStart,
+  touchMove,
+  touchRelease,
+} from 'helper';
+
+import TouchSensor from './..';
```

Comment:
woa what is this doing? It looks at the current directory but _then_ goes up a level and the filename is implicit?

Reply:
Yeah, it will use the `index.js` of that directory. This is on purpose, I want to test what gets ultimately exported, otherwise we could do `import TouchSensor from './../TouchSensor'`, but that won't export everything that we may expect, if that makes sense